### ISSUE TITLE
Broken Source Link

### ISSUE THREAD

**Issue by Clari-7744** [FemArch/Website#1]
On the website, there's a "Source" link, that leads to https://github.com/FemArch/FemArch, which is a 404.
Should it be https://github.com/FemArch/fem_repo?

**Comment by Mcharlsto** [comment]
Yeah, you're right. The link's correct, but the repo is private so it won't be publicly accessible. I'll update it when I'm back at my PC.

Honestly mostly surprised someone's found this mess, just be aware that the ISO doesn't work and half the links don't go anywhere. Have fun 😭

**Comment by Clari-7744** [comment]
oh lol

lmao, my friend contributed to and starred the website so it showed up on
my front page
